src/webpack/tagCommonJSExports.js

```diff
@@ -1,4 +1,6 @@
-;(function () {
+/* global __FILENAME__ */
```

Comment:
Kinda silly to leak these into the generated code. Can we add some naïve code to cut comments in the Webpack loader?